lib/initializer.rb

```diff
@@ -0,0 +1,47 @@
+require_relative './game.rb'
+require_relative './board/board.rb'
+require_relative './board/board_checker.rb'
+require_relative './players/ai.rb'
+require_relative './players/human.rb'
+require_relative './players/player.rb'
+require_relative './ui/user_interface.rb'
+require_relative './ui/script.rb'
+
+def initializer(size, empty_char)
+  board = Board.new(size, empty_char)
+  checker = BoardChecker.new(size, empty_char)
```

Comment:
Seems strange to me that the board & the board checker take the same arguments to be initialized, is there any way to clean this up? Maybe passing a board to the methods that do the checking?